unittests/AST/ASTImporterTest.cpp

```diff
@@ -11,6 +11,8 @@
 //
 //===----------------------------------------------------------------------===//
 
+#define GTEST_HAS_COMBINE 1
```

Comment:
Why is this needed? Because of the multiple inheritance of the test fixtures? Could you please add a comment in the code?

Reply:
I do not like this `#define` but could not find other fast solution to make it work (without this define `Combine` is not avaliable).